src/api/app/models/canned_response.rb

```diff
@@ -15,7 +15,8 @@ class CannedResponse < ApplicationRecord
   enum decision_type: {
     cleared: 0,
     favored: 1,
-    favored_with_comment_moderation: 2
+    favored_with_comment_moderation: 2,
+    favored_delete_request: 3
```

Comment:
I'd propose to follow the same naming convention:
```suggestion
    favored_with_delete_request: 3
```

Reply:
True, changed!